modules/ROOT/pages/errors/gql-errors.adoc

```diff
@@ -1403,6 +1403,18 @@ Status description:: error: system configuration or operation exception - bolt i
 
 Status description:: error: system configuration or operation exception - unsupported operation of a sharded database. Feature: `{ $feat }` is not available in a sharded database.
 
+=== 51N72
+
+Status description:: error: system configuration or operation exception - memory pool out of memory. Failed to allocate memory in a memory pool. See `{ $cfgSetting }` in the neo4j configuration.
+
+=== 51N73
+
+Status description:: error: system configuration or operation exception - transaction memory limit reached. The transaction used more memory than was allowed. The maximum allowed size for a transaction can be configured with `{ $cfgSetting }` in the neo4j configuration.
```

Comment:
```suggestion
Status description:: error: system configuration or operation exception - transaction memory limit reached. The transaction uses more memory than it is allowed. The maximum allowed size for a transaction can be configured with `{ $cfgSetting }` in the neo4j.conf file.
```

Reply:
We try to use present tense whenever possible. I don't know if this message has already been merged in the monorepo.